sdk/config_helpers.go

```diff
@@ -268,7 +268,7 @@ func updateNginxConfigFileConfig(
 				if err := updateNginxConfigFileWithRoot(aux, directive.Args[0], seen, allowedDirectories, directoryMap); err != nil {
 					return true, err
 				}
-			case "ssl_certificate", "proxy_ssl_certificate", "ssl_client_certificate", "ssl_trusted_certificate":
+			case "ssl_certificate", "proxy_ssl_certificate", "ssl_client_certificate", "ssl_trusted_certificate", "ssl_certificate_key":
```

Comment:
we intentionally don't upload "ssl_certificate_key" because we don't want to send the customer private keys over the config payload. Can you remove this change please?

Reply:
Hello,
First of all - thank you for you feedback, it is very appreciated!
Regarding the code - yes, I will remove unnecessary parts.